cfme/tests/services/test_provision_stack.py

```diff
@@ -154,9 +155,10 @@ def _cleanup(appliance=None, provision_request=None, service=None):
     else:
         myservice = service
     if myservice.exists:
-        myservice.delete()
+        myservice.retire()
```

Comment:
@jaryn do we want to retire then delete the service? Or only retire the service? Won't it stay in the inventory if it is only retired?

Reply:
it was not so easy as I though, but done